doctrine/phpcr-bundle/2.0/config/packages/doctrine_phpcr.yaml

```diff
@@ -0,0 +1,13 @@
+# Read the documentation: https://symfony.com/doc/2.0/cmf/bundles/phpcr_odm/configuration.html
+# Configure a doctrine dbal default connection before
+#doctrine_phpcr:
+#    odm:
+#        auto_mapping: true
+#        auto_generate_proxy_classes: "%kernel.debug%"
+#    session:
+#        backend:
+#            type: doctrinedbal
+#            connection: default
+#        workspace: default
+#        username: '%env(PHPCR_SESSION_USERNAME)%'
+#        password: '%env(PHPCR_SESSION_PASSWORD)%'
```

Comment:
We do not copy/paste the default configuration as it means syncing it whenever something change upstream. Instead, we only list the ones which needs to be changed by the end user. If there is none, then the file can be removed.

Reply:
Mh, ok sounds valid. But then i would have a question/feature request: is it possible to use env vars as default on configuration directly? I mean would `->defaultValue( '%env(PHPCR_SESSION_USERNAME)%')` work? Or shouldn't we have an extra `defaultEnvironmentValue('PHPCR_SESSION_USERNAME', optonalCast)` for that?

Reply:
The bundle should not handle environment variables. The bundle should only care about its config. 

When defining your config in `Configuration.php` you should use `->defaultValue('foobar')`. When the user is using the config in `config.yaml` they might want to use `%env(FOO)%`.

Reply:
So providing a configuration for both credential parts (username, password) set by envirionment variables here would be a good example?
Then i would remove the other configuration parts. But i think i would keep the decission for `doctrinedbal` and add a hint, that there is the place to switch to an other backend, and that dbal would need doctrine dbal configuration. Doing so we would have one working exampe then.